docs/tutorials/fairplay.md

```diff
@@ -61,3 +61,43 @@ player.getNetworkingEngine().registerResponseFilter((type, response) => {
   response.data = shaka.util.Uint8ArrayUtils.fromBase64(responseText).buffer;
 });
 ```
+
+## Integration with some DRMs providers
```

Comment:
I feel like these can be sub-sections of the previous section (`## License wrapping`), instead of having a new section.
That section already mentions the idea that some FairPlay servers need different formats, so these sub-sections would make sense inside it, I feel like.

Incidentally, looking at that previous section, I just noticed it advises writing SPC request and response filters manually when the user could just be using `FairPlayUtils.spcFairPlayRequest` and `FairPlayUtils.commonFairPlayResponse`. I can see the logic in showing how writing a filter actually works, but it seems odd to not mention the premade ones.